README.md

```diff
@@ -1,4 +1,25 @@
-# NCEPLIBS-ncio library
+
+# NCEPLIBS-ncio
+
+This is a library used by NCEP models for netCDF I/O. This is part of
+the [NCEPLIBS](https://github.com/NOAA-EMC/NCEPLIBS) project.
+
+## Authors
+
+Jeff Whitaker, Rahul Mahajan
```

Comment:
@aerorahul I'm giving you credit as an author. If that's incorrect, or you don't want me to, then let me know.

Reply:
I am not the author.  Jeff Whitaker and Cory Martin are.

Reply:
OK thanks fixed.